src/routers/projects.js

```diff
@@ -2,8 +2,91 @@ const express = require("express");
 
 const router = express.Router();
 
-router.get("/", function (req, res, next) {
+router.get("/", function (req, res) {
   res.status(200).send({ sucess: true });
 });
 
+router.get("/:projectId/versions", async (req, res) => {
+  const { projectId } = req.params;
+  const accessToken = req.headers.authorization;
+
+  if (accessToken) {
+    try {
+      const getVersions = await fetch(
+        `https://api.figma.com/v1/files/${projectId}/versions`,
+        {
+          method: "GET",
+          headers: {
+            "X-FIGMA-TOKEN": accessToken,
+          },
+        },
+      );
+
+      const responseJson = await getVersions.json();
+      const { status, versions } = responseJson;
+
+      if (status === 400) {
+        return res.status(400).send({
+          result: "error",
+          error: {
+            message: "Bad Request",
+          },
+        });
+      }
+
+      if (status === 403) {
+        return res.status(403).send({
+          result: "error",
+          error: {
+            message: "인증되지 않은 사용자 접근입니다.",
+          },
+        });
+      }
+
+      if (status === 404) {
+        return res.status(404).send({
+          result: "error",
+          error: {
+            message: "존재하지 않는 피그마 파일입니다. 다시 입력해주세요",
+          },
+        });
+      }
+
+      if (status === 500) {
+        return res.status(500).send({
+          result: "error",
+          error: {
+            message: "프로젝트 용량이 너무 커 응답에 실패했습니다.",
+          },
+        });
+      }
+
+      if (versions.length < 2) {
+        return res.status(502).send({
+          result: "error",
+          error: {
+            message: "비교할 수 있는 버전이 없어서 서비스 이용이 불가합니다.",
+          },
+        });
+      }
+
+      return res.status(200).send(responseJson);
+    } catch (e) {
+      return res.status(500).send({
+        result: "error",
+        error: {
+          message: "Internal Server Error",
```

Comment:
P3: 에러 메세지 형태는 한글/영어 통일해갔으면 좋겠습니다.
경준님이 작성하신 형태처럼 한글로 가져가는 게 좋은 듯한데 경준님 생각은 어떠신가요? 
성호님도 같이 보면 좋을 것 같습니다! @tjd985 

```suggestion
          message: "시스템 장애로 통신이 원활하지 않습니다.",
```
<br />

[토스가 에러 메세지를 전달하는 방식](https://toss.im/tosscert/docs/guides/errorcode)

Reply:
좋은 것 같습니다!